### DISCUSSION TITLE
Release schedule for opentelemetry-ruby

### DISCUSSION THREAD

**Discussion by chrisholmes** [open-telemetry/opentelemetry-ruby#1196] [General]
Are there any plans to adopt a regular release schedule or similar commitments for the Otel-ruby gems?

There have been many useful additions and fixes to the project since the last wide release that I would be keen to adopt, but we haven't seen any releases since February.

**Comment by arielvalentin** [comment]
Thank you for your patience. 

We have had limited capacity and have been focusing our effort on extracting various gems into the contrib repository. 

If there are any gems that you are waiting on a release for please respond to this thread and I will request a release from the @open-telemetry/ruby-maintainers and hopefully they will be released as soon as possible.

**Reply by chrisholmes** [reply]
Thanks for the quick reply @arielvalentin. Are there any areas I could provide contributions or support while capacity is low?

As for releases, I think the following gems have changes that should be released:
- `opentelemetry-instrumentation-actionpack`
- `opentelemetry-instrumentation-rails`
- `opentelemetry-instrumentation-rdkafka` (this might require a release of all?)
- `opentelemetry-instrumentation-faraday`
- `opentelemetry-exporter-otlp`

**Reply by arielvalentin** [reply]
Here is the release request https://github.com/open-telemetry/opentelemetry-ruby/pull/1197

Yes! We can always use help!!! But as always my terrible project management skills make it so you cannot tell what I need help with. i will work on triaging a list and getting it ready for the next SIG meeting on Tuesday and CC you on the issue.

**Reply by arielvalentin** [reply]
Then again it occurs to me that maybe you'd be able (and willing) to help us define a release schedule?

**Reply by chrisholmes** [reply]
Sure, I’ll gather some research on how this is handled elsewhere to start with.
